ckanext/fjelltopp_theme/assets/css/fjelltopp-theme.scss

```diff
@@ -1611,11 +1611,12 @@ a.thumbnail.active {
 
 .media-grid .media-image {
     width: auto;
-    aspect-ratio: 9 / 4;
+    aspect-ratio: 16/ 9;
```

Comment:
Why change AR?

Reply:
Put back to previous AR, logo is a bit smaller though.